src/current/molt/migrate-in-phases.md

```diff
@@ -0,0 +1,158 @@
+
+{% include molt/migration-prepare-database.md %}
+
+## Step 2. Prepare the source schema
+
+{% include molt/migration-prepare-schema.md %}
+
+## Step 3. Load data into CockroachDB
+
+{{site.data.alerts.callout_success}}
+To optimize performance of [data load](#step-3-load-data-into-cockroachdb), Cockroach Labs recommends dropping any [constraints]({% link {{ site.current_cloud_version }}/alter-table.md %}#drop-constraint) and [indexes]({% link {{site.current_cloud_version}}/drop-index.md %}) on the target CockroachDB database. You can [recreate them after the data is loaded](#step-5-modify-the-cockroachdb-schema).
+{{site.data.alerts.end}}
+
+Perform an initial load of data into the target database. This can be a subset of the source data that you wish to verify, or it can be the entire dataset.
+
+{% include molt/fetch-data-load-modes.md %}
+
+1. Issue the [MOLT Fetch]({% link molt/molt-fetch.md %}) command to move the source data to CockroachDB, specifying `--mode data-load` to perform a one-time data load. For details on this mode, refer to the [MOLT Fetch]({% link molt/molt-fetch.md %}#load-data) page.
+
+	{{site.data.alerts.callout_info}}
+	Ensure that the `--source` and `--target` [connection strings]({% link molt/molt-fetch.md %}#connection-strings) are URL-encoded.
+	{{site.data.alerts.end}}
+
+	<section class="filter-content" markdown="1" data-scope="postgres">
+	{% include_cached copy-clipboard.html %}
+	Be sure to specify `--pglogical-replication-slot-name`, which is required for replication in [Step 6](#step-6-replicate-changes-to-cockroachdb).
+
+	~~~ shell
+	molt fetch \
+	--source 'postgres://postgres:postgres@localhost:5432/molt?sslmode=verify-full' \
+	--target 'postgres://root@localhost:26257/defaultdb?sslmode=verify-full' \
+	--table-filter 'employees' \
+	--bucket-path 's3://molt-test' \
+	--table-handling truncate-if-exists \
+	--non-interactive \
+	--pglogical-replication-slot-name cdc_slot \
+	--mode data-load
+	~~~
+	</section>
+
+	<section class="filter-content" markdown="1" data-scope="mysql">
+	{% include_cached copy-clipboard.html %}
+	~~~ shell
+	molt fetch \
+	--source 'mysql://user:password@localhost/molt?sslcert=.%2fsource_certs%2fclient.root.crt&sslkey=.%2fsource_certs%2fclient.root.key&sslmode=verify-full&sslrootcert=.%2fsource_certs%2fca.crt' \
+	--target 'postgres://root@localhost:26257/defaultdb?sslmode=verify-full' \
+	--table-filter 'employees' \
+	--bucket-path 's3://molt-test' \
+	--table-handling truncate-if-exists \
+	--non-interactive \
+	--mode data-load
+	~~~
+	</section>
+
+{% include molt/fetch-data-load-output.md %}
+
+## Step 4. Verify the data load
+
+{% include molt/verify-output.md %}
+
+Repeat [Step 3](#step-3-load-data-into-cockroachdb) and [Step 4](#step-4-verify-the-data-load) to migrate any remaining tables.
+
+## Step 5. Modify the CockroachDB schema
+
+{% include molt/migration-modify-target-schema.md %}
+
+## Step 6. Replicate changes to CockroachDB
```

Comment:
@rohan-joshi @Jeremyyang920 should we add the section about how to verify replication has caught up and the next step (cutover) can happen?

Reply:
`fetch-replication-output.md`, which included in this step, has this information:

```
`upserted rows` log messages indicate that changes were replicated to CockroachDB:
```

Is that sufficient or should we enhance/update it?

Reply:
There are also metrics that you can look at if you enabled the metrics endpoint.

Reply:
Which metrics?

Reply:
"expr": "sum by (schema) (rate(apply_upserts_total[$__rate_interval]) + rate(apply_deletes_total[$__rate_interval]))",

You can use the apply_upserts_total, and apply_deletes total, to calculate the rate of change over time and ovserve that number to go to 0 meaning things should be caught up.